vyper/old_codegen/lll_node.py

```diff
@@ -92,6 +92,10 @@ def __init__(
         self.total_gas = None
         self.func_name = None
 
+        def _check(condition, err):
+            if not condition:
+                raise CompilerPanic(str(err))
```

Comment:
Really not saving all that much code, and probably makes it harder to read when a panic does happen